connectivity/suite.go

```diff
@@ -197,6 +212,64 @@ func Run(ctx context.Context, ct *check.ConnectivityTest) error {
 			return check.ResultOK, check.ResultDrop
 		})
 
+	// This policy denies all ingresses by default
+	ct.NewTest("all-ingress-deny").
+		WithPolicy(denyAllIngressPolicyYAML).
+		WithScenarios(
+			tests.PodToPod(),
+			tests.PodToCIDR(), // This should be allowed by the policy
```

Comment:
It would be more explicit to lay this out further:
- Pod to Pod fails because there is no egress policy (so that part works) but then at the destination there is ingress policy that denies the traffic.
- Egress to world works because there is no egress policy, then when replies come back, they are considered as "replies" to the outbound connection so they are not subject to ingress policy.

Reply:
agreed :100: 

There is a related PR to make sure that we will verify if the traffic is dropped due to which direction (e.g. Ingress or Egress)

https://github.com/cilium/cilium-cli/pull/1095